rumcake/src/hw/mod.rs

```diff
@@ -123,6 +127,22 @@ pub async fn output_switcher() {
     }
 }
 
+#[cfg(feature = "bootloader-double-tap-reset")]
+pub static BOOTLOADER_MAGIC: u32 = 0xDEADBEEF;
+
+#[cfg(feature = "bootloader-double-tap-reset")]
+#[link_section = ".uninit"]
+pub static mut FLAG: GroundedCell<u32> = GroundedCell::uninit();
```

Comment:
At first I just used `MaybeUninit` and that did not work. I [asked](https://matrix.to/#/!YoLPkieCYHGzdjUhOK:matrix.org/$GTTNZqyZ9Jq2zKf8N1CfKx-ytSRkgaO5xo9APNFd1lY?via=matrix.netflam.de&via=matrix.org&via=tchncs.de) in the embassy matrix channel for help and they said that it would need to be wrapped in an `UnsafeCell` because the compiler can otherwise make assumptions on the `MaybeUninit`. I now tested it again with just `MaybeUninit` and it works, so now I am unsure if the extra `UnsafeCell` is even needed. Either way, the grounded crate is just a wrapper around `UnsafeCell<MaybeUninit<T>>` so it can be replaced with that anyways to remove that extra dependency.

Reply:
to be safe I just changed it to `UnsafeCell<MaybeUninit<T>>` now.

Reply:
Thanks, I'll test it on my hardware as well to verify.